src/reducers/accountCreator.js

```diff
@@ -0,0 +1,63 @@
+import {combineReducers} from 'redux';
+import {
+  GENERATE_NEW_KEYPAIR,
+  UPDATE_FRIENDBOT_TARGET,
+  START_FRIENDBOT_REQUEST,
+  FINISH_FRIENDBOT_REQUEST,
+} from '../actions/accountCreator';
+import _ from 'lodash';
+
+function keypairGeneratorResult(state=null, action) {
+  if (action.type === GENERATE_NEW_KEYPAIR) {
+    return {
+      pubKey: action.pubKey,
+      secretKey: action.secretKey,
+    };
+  }
+  return state;
+}
+function keypairGeneratorPubKey(state='', action) {
+  if (action.type === GENERATE_NEW_KEYPAIR) {
+    return action.pubKey;
+  }
+  return state;
+}
+function friendbotTarget(state='', action) {
+  if (action.type === UPDATE_FRIENDBOT_TARGET) {
+    return action.target;
+  }
+  return state;
+}
+
+const defaultRequestState = {
+  message: '',
+  code: '',
+  status: 'inital',
+}
+function friendbotStatus(state, action) {
+  if (state === undefined) {
```

Comment:
This seems like a mistake...  If the first time this reducer is run you entirely ignore the action?  Could you explain the design decision's here?